### ISSUE TITLE
Double tooltips using SVG renderer and vega-tooltip.

### ISSUE THREAD

**Issue by vorbei** [vega/vega#3148]
In Observable, like: https://observablehq.com/@vega/vega-lite-api-v5,
add a cell with code like

```javascript
  vl
    .markBar()
    .encode(vl.y().fieldQ('a'), vl.x().fieldN('i'), vl.tooltip().field('a'))
    .data([{ a: 1, i: 0 }, { a: 2, i: 1 }, { a: 3, i: 2 }])
    .render({renderer:'svg'})
```

Hovering on the bars, first the vega-lite tooltip appears, then after several seconds the browser native tooltip will appear.

The 'canvas'/'png' renderer doesn't have the issue, and vega-lite-api@4 doesn't have this issue either.
(In notebook https://observablehq.com/@vega/vega-lite-api)

**Comment by jheer** [comment]
This [online editor example](https://vega.github.io/editor/#/url/vega/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykSArJQBWENgDsQAGhBMkUANZkATmwbiAJmhAB3GHTjSQOJBo01xZNAJnwaZLGgBMABhcyImAJ4AbQ+ig4Hx8jDSRMFFQAbVBxJAR-EDEGZUCAfRcjYiQfBjgINBiQSIBGGRo0FwBfKVBIp3K0Epq6tABmRtQnKoBdFpA4hO0wiIyjZNTEifTMmUxlJHEIADM2ZQRC0G8cKYjFIxU1HCYvQpAKnpllmiCtdBQPNcw0UGvbwsuQNY04ZQ+a4oFaLFNKeJDKZ4yJBpOCaECfNjLZYQODPdAAL1+bBA-W2iWuPkwvyMcAAHjg-ugaBAAGo5GgaAAUIwYCCiAB1ipyegBKAAEADIBXzqQAxCwGRkAahZbM5SG5PJxfU+EHscR8QKKg0SpNBRJwWRyeWc1VqA3iiR0DPoRgYOBGiTkmggJkCzLYyAsjIA5KSfTzKH5LPQpHyXJQyuHKC4BPyAFR8vWeOCG3oyZDKBRa2KW7SZ7NGPHaZRwKCQpLePxnOR-T7LVQbVCgEaRECtsYA2FQNg-F4ge2O-sEkLNkDZXKJADEABYoAB2AAcSEXOLmbDYhJohrHarIGu01LpPgZzPCrI5XPh-IA-HzZZeFfC+ag+ZzOTLz3Krz018VlDQSAAEqbnAAAi+RQABOCYOwkhjhOJroLWf4-NA0GwRI-Z7ge6Dyq+nJ8lKfKMqs6zhGemAXvK3Jhu+IA8vyxGcgA3CKBEgERJFHvSTIPpyNCKnyd78ec3Ivm+IAfqJgnXn+pLYVAOS6kYbw+Hc5x-taGi2ruSnVugCkyM6dzNDIpx6cp2inFcNzqdo0KwloAJeE4ilWegNkgGpGnQmCEI4lUQWqvp+SbBaQyGUWXg7NoJmhJ6SAWP2rbDOE0KzN5dkaRUjwBag8x5ACCyWP4oApju5Xqjk2jJgagUyCYZgWGQACS4jiMSqARmUximOYlgAPIMESlIRrG-Q6tZ0WxegJ6deCCVevBLbpWloyZT5Wogv5FaOXCfQyCVZD+EUOE1egdgOM8NQuJ84g0IEaCFXAMiYqoz3KEVnxIKSYXROVoXTTIaw3OIaIgH4ywVioDIAMpA1FMiw3cL1zI9CgAMJqOD2HVaOICBDQPiMldWAEDOLhKgCGiJclqDLDkKIyD4SBMEEQKM5qr3-oBaBc8zIDIKSACipJErj3UZhYYsS2i7ggLBiic0zPPohYPwKd1-TQB5IBGV8AGwhDTBsNgnoHABdwCzzsGYAZmks2zQQAIInvuJb2I4Tvsz4LuWAZTjzgrrO+wAQkgKLzYkCAMho1ZvRrZKVDriMgF5oPG9oUMw1b-Oq+j9uJA8kPOz4g2kMorM7mjisY9j6holV+4XYTcDE6T7fXRTVN-urmjJ9rfRAA) has the compiled Vega for the API example above. The issue appears in Vega v5.20+, but not in v5.19.1.

**Comment by jheer** [comment]
Fixed in the v5.20.2 release.
